.github/workflows/pull-fork.yml

```diff
@@ -0,0 +1,89 @@
+name: Pull request build
+
+on:
+  pull_request:
+    types:
+      - opened
+      - synchronize
+
+defaults:
+  run:
+    working-directory: src/client
+
+env:
+  BUCKET_WEB: reactive-trader-web-builds
+  BUCKET_OPENFIN: reactive-trader-openfin-builds
+  BUCKET_FINSEMBLE: reactive-trader-finsemble-builds
+
+jobs:
+  build:
+    name: Build & deploy
+
+    runs-on: ubuntu-20.04
+
+    if: "github.event.pull_request.header.repo.fork"
```

Comment:
according to my (admittedly not exhaustive/empirical) research, this should be
```
 if: "github.event.pull_request.head.repo.fork"
```
note "head" not "header"

Also, this will need to be updated in the `pull.yml` workflow as it was incorrect there in the first place